### ISSUE TITLE
Add Corner Radius Config Option for Bar Chart

### ISSUE THREAD

**Issue by GeorgeBerdovskiy** [PureChart/purechart#39]
We currently support some very nice bar charts, but styling options are quite limited. This goes against our goal of complete control over chart appearance! Let's work towards that goal by adding a configuration option that specifies how rounded the corners should be. Check out the image below for guidance.

<p align="center">
<img width="1202" alt="image" src="https://github.com/PureChart/purechart/assets/43429820/be962c37-8047-46a2-8dcf-c1b062c6bf6f">
</p>

Right now, the configuration argument looks like this -
```rb
@configuration = {
  axes: {
    horizontal: "Lines of JavaScript",
  }
}
```

Change the `bar_chart` helper code to accept arguments that also look like this -
```rb
@configuration = {
  axes: {
    horizontal: "Lines of JavaScript",
  },
  corner_radius: "5px"
}
```

However, remember that this should be an _option_! In other words, if the user doesn't specify a corner radius, use some default. **Smooth sailing ⛵️**

**Comment by sarayumummidi** [comment]
I can take a shot at this!

**Comment by GeorgeBerdovskiy** [comment]
Closed by #43
